pkg/pci/pci.go

```diff
@@ -27,7 +27,7 @@ type PCI struct {
 
 	Latency   byte
 	IRQPin    byte
-	IRQLine   uint8
+	IRQLine   uint32
```

Comment:
per discussions with Linux kernel expert, this should be a uint. The idea of "IRQLine" means little or nothing in an age when a single device can have 2048 interrupts.